### ISSUE TITLE
Calendar does not render on Apple iPad 3 (iOS 9.3.5), "Can't find variable: Intl"

### ISSUE THREAD

**Issue by badger-github** [fullcalendar/fullcalendar#4558]
Hi

Have ported an app from version 3 to version 4 for testing, when using an iPad 3 (which is limited to iOS 9.3.5), full calendar does not render at all (blank page).  

Tested / Recreated using the demo v4 page from FullCalendar’s web site: https://fullcalendar.io/docs/v4/initialize-globals-demo

Error in Inspector is - "Can't find variable: Intl"

I googled and found a solution by adding the following script tag:

<script src="https://polyfill.io/v3/polyfill.min.js?features=Intl.~locale.en"></script>

Calendar then renders correctly and drag/drop/eventclick all work from my limited testing.

Unfortunately the formatDate function no longer works:

FullCalendar.formatDate(info.event.start, { month: '2-digit', year: 'numeric', day: '2-digit' })

Event Triggers for Month and Day are sent as single digits if they are in 1-9 range

Before I modify my app I was just wondering if this is a supported platform / os for version 4 and if there was a better workaround that would allow formatDate to work.

Thank you for your time, this is an excellent project.

Pete

**Comment by arshaw** [comment]
Hey Pete, that's a clever ideal to use the polyfill for the Intl api.

I'm having trouble understanding your exact use of `formatDate`. So, would you be able to post a [runnable, stripped-down demonstration](http://fullcalendar.io/wiki/Reduced-Test-Cases/)? It's the only way I can help.

**Comment by badger-github** [comment]
Hi
Thank you for replying to this, after more investigating it would seem iOS9 is abandonware, which I am a bit chuffed off about, but that's a different issue.  I cannot use jsfiddle or codepen on iOS9 so here is an example on my site. 

https://ovits.com.au/dt.html

This DOES NOT include any Fullcalendar code so no longer an issue for this project.  But if you are interested on iOS12.1.4 and Safari (12.0.3) this produces output "21/03/19" on iOS9.3.5 this produces output "3/21/19".  Same Date/Time settings / Regional Options.

The way I was using dates in my app is unique/bad and my own issue so I just worked around this problem.

Apologies if I wasted your time.

Thanks again Pete
